### PR TITLE
Add watermark-conference-for-women

### PR DESCRIPTION
**Pull request by fabricioblz**
https://github.com/cristianoliveira/awesome4girls/issues/108

It's sold out, should we publish anyway?

I've tried to adjust the text for a less marketing text (Open to suggestions)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -316,6 +316,10 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 
     The Society of Women Engineers (SWE) is a not-for-profit educational and service organization that empowers women to succeed and advance in the field of engineering and to be recognized for their life-changing contributions as engineers and leaders. It promotes the annual conference for women engineers.
 
+  - [Watermark Conference for Women](http://www.watermarkconferenceforwomen.org/conference/) - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/a/a4/Flag_of_the_United_States.svg" alt="United States" width="30">
```
Comment (cristianoliveira) [inline README.md]: Please point to their site instead:
[Watermark Conference for Women](http://www.watermarkconferenceforwomen.org/)

File: README.md [hunk 2]
```diff
@@ -316,6 +316,10 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
...
+
+    The conference features nationally speakers who share their wisdom and expertise on a wide range of personal and professional development topics.
+
```
Comment (cristianoliveira) [inline README.md]: How about this? Got directly from their website:

"The mission of the Conference for Women is to promote, communicate and amplify the influence of women in the workplace and beyond. At the annual nonpartisan, nonprofit conferences, it brings together thousands of active professionals to connect, renew, and find inspiration in the community."

Comment (harsh183) [reply README.md]: That seems quite good yes!

**Comment by harsh183** [comment]
@fabricioblz Assuming it runs every year, I think we can keep it.

File: README.md (hunk 2 above)
Comment (fabricioblz) [reply README.md]: Good!

Done!

File: README.md (hunk 1 above)
Comment (fabricioblz) [reply README.md]: Didn't get that...

The link is wrong?

**Comment by cristianoliveira** [comment]
Please keep the list in alphabetical order.

Expected order:
"alterconf", "ela conf", "european women in technology", "girls in tech catalyst conference", "global tech women's annual voices conference", "grace hopper celebration of women in computing", "johnson women in technology conference", "lesbians who tech", "she's geeky", "shesharp", "swe", **"tech up for women", "watermark conference for women", "we rise",** "wecode", "witi", "women in tech summit", "women in tech summit series", "women of the channel leadership summit west", "women who tech", "write/speak/code"

**Comment by fabricioblz** [comment]
Order Task Done!

Is there new specs?  Some new things start to broke

**Comment by cristianoliveira** [comment]
Hey @fabricioblz something went really wrong out there hahahah
Let me take a look here and point what do you need to change

**Comment by cristianoliveira** [comment]
Hey @fabricioblz turned out was just a matter of rebasing here. I merged your PR already :)

Thanks for your contribution and patience :)

**Comment by fabricioblz** [comment]
You Welcome @cristianoliveira!

**Pull request merged**